README.md

```diff
@@ -0,0 +1,44 @@
+[![CRAN_Status_Badge](http://www.r-pkg.org/badges/version/text2vec)](http://cran.r-project.org/package=text2vec)
+[![Travis-CI Build Status](https://travis-ci.org/dselivanov/text2vec.svg?branch=master)](https://travis-ci.org/dselivanov/text2vec)
+[![License](http://img.shields.io/:license-mit-blue.svg?style=flat)](http://badges.mit-license.org)
+[![Downloads](http://cranlogs.r-pkg.org/badges/grand-total/text2vec)](http://cran.rstudio.com/package=text2vec)
+[![Follow](https://img.shields.io/twitter/follow/dselvan0v.svg?style=social)](https://twitter.com/intent/follow?screen_name=dselvan0v)
+
+# Tutorials
```

Comment:
We can replace links with CRAN-based html version of vignettes (as you suggested):
[GloVe](https://cran.r-project.org/web/packages/text2vec/vignettes/glove.html), [vectorization](https://cran.r-project.org/web/packages/text2vec/vignettes/text-vectorization.html)
Also will be nice to add link to articles marked with [text2vec tag](http://dsnotes.com/tags/text2vec/).